src/abi.h

```diff
@@ -327,4 +327,43 @@ struct netplan_net_definition {
     /* netplan-feature: eswitch-mode */
     char* embedded_switch_mode;
     gboolean sriov_delay_virtual_functions_rebind;
+
+    /* vxlan */
```

Comment:
```suggestion
    /* vxlan */
    /* netplan-feature: vxlan */
```

Please let's add a feature flag here for the new VXLAN feature. Also, maybe move the VXLAN fields below the new VRF fields, so they are listed together with the `vxlan_params` and not intertwined.

Reply:
Done.